pandas-stubs/core/series.pyi

```diff
@@ -1640,6 +1651,9 @@ class Series(IndexOpsMixin[S1], NDFrame):
         self, other: int | np_ndarray_anyint | Series[int]
     ) -> Series[int]: ...
     def __rsub__(self, other: num | _ListLike | Series[S1]) -> Series: ...
+    @overload
+    def __rtruediv__(self, other: Path) -> Series: ...
```

Comment:
I think this overload could be combined with the next one

Reply:
fixed in next commit